core/market/decentralized/tests/test_agreement.rs

```diff
@@ -21,7 +22,7 @@ const PROV_NAME: &str = "Node-2";
 #[actix_rt::test]
 #[serial_test::serial]
 async fn test_gsb_get_agreement() -> Result<()> {
-    let network = MarketsNetwork::new("test_gsb_get_agreement")
+    let network = MarketsNetwork::new(generate_backtraced_name().as_str())
```

Comment:
Don't we rather want it to be invoked from within the `MarketsNetwork`? It will then be totally transparent for future readers and writers of the new tests. I believe it is worth doing.
```suggestion
    let network = MarketsNetwork::new()
```